easybuild/easyblocks/n/namd.py

```diff
@@ -131,8 +151,12 @@ def test_step(self):
             namdcmd = os.path.join(self.cfg['start_dir'], self.namd_arch, 'namd%s' % self.version.split('.')[0])
             if self.cfg['charm_arch'].startswith('mpi'):
                 namdcmd = self.toolchain.mpi_cmd_for(namdcmd, 2)
-            cmd = "%(namd)s %(testdir)s" % {
+            ppn = ''
+            if self.toolchain.options.get('openmp', False):
+                ppn = '+ppn 2'
```

Comment:
This is the number of threads to use, and since we're testing openmp, in this case, it's better to turn it on and set to 2 than let it fly off on it's own and try to use #cores as threads.